cmd/mimir/config-descriptor.json

```
@@ -10063,7 +10063,7 @@
               "kind": "field",
               "name": "enabled",
               "required": false,
-              "desc": "Enable running rule groups against multiple tenants. The tenant IDs involved need to be in the rule group's 'source_tenants' field. If this flag is set to 'false' when there are already created federated rule groups, then these rules groups will be skipped during evaluations.",
+              "desc": "Enable rule groups querying against multiple tenants. The tenant IDs involved need to be in the rule group's 'source_tenants' field. If this flag is set to 'false' when there are already created federated rule groups, then these rules groups will be skipped during evaluations.",
```

```suggestion
              "desc": "Enable rule groups to query against multiple tenants. The tenant IDs involved need to be in the rule group's 'source_tenants' field. If this flag is set to 'false' when there are federated rule groups that already exist, then these rules groups will be skipped during evaluations.",
```
What does "against" mean here? Does it mean, "Enable rule groups to perform queries on multiple tenants."?

---

> What does "against" mean here? Does it mean, "Enable rule groups to perform queries on multiple tenants."?

Your suggestion has it right 👍